src/components/AppBar.vue

```diff
@@ -0,0 +1,98 @@
+<script setup>
+import { WalletMultiButton } from 'solana-wallets-vue'
+import store from '../vuex'
+
+
+function flipAutoConnect (balance) {
+  store.commit('setAutoConnect')
+}
+</script>
+
+<template>
+  <div>
+
+    <!-- NavBar / Header -->
+    <div class="navbar flex flex-row md:mb-2 shadow-lg bg-neutral text-neutral-content">
+      <div className="navbar-start">
```

Comment:
Any reason we are using `className` here or is this copy/paste from React that hasn't been updated haha?

(Same goes on other DOM elements and other components btw)

Reply:
oops! wasn't aware there was a difference thanks for pointing this out